tarantool_test.go

```diff
@@ -352,6 +356,52 @@ func BenchmarkClientParallelMassiveUntyped(b *testing.B) {
 	close(limit)
 }
 
+func BenchmarkConnection_Replace(b *testing.B) {
+	conn, err := Connect(server, opts)
+	if err != nil {
+		b.Errorf("No connection available")
+		return
+	}
+	defer conn.Close()
+	spaceNo = 520
+
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			_, err := conn.Replace(spaceNo, []interface{}{uint(1), "hello", []interface{}{}})
+			if err != nil {
+				b.Log(err)
+				b.Errorf("No connection available")
+			}
+		}
+	})
+}
+
+func BenchmarkConnection_LargeSelect(b *testing.B) {
+	conn, err := Connect(server, opts)
+	if err != nil {
+		b.Errorf("No connection available")
+		return
+	}
+	defer conn.Close()
+
+	schema := conn.Schema
+	rSpaceNo, rIndexNo, err := schema.ResolveSpaceIndex("test_perf", "secondary")
+	if err != nil {
+		b.Errorf("symbolic space and index params not resolved")
+	}
+
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			_, err := conn.Select(rSpaceNo, rIndexNo, 0, 10000000, IterEq, []interface{}{"test_name"})
```

Comment:
Also, please put request limit (and maybe an offset too) in a separate variable so it would be easier to read this code. I've needed to go into doc to check what fourth argument is.

Reply:
Done.